lang/pt-BR.js

```diff
@@ -1,3 +1,83 @@
 export default {
```

Comment:
Again about including this file - or is there a list somewhere of what needs to be translated?

Reply:
I've already pinged the corresponding contributors, we may have to keep few limits / guidelines for i18n in near future.